### DISCUSSION TITLE
Can't access from outside

### DISCUSSION THREAD

**Discussion by Azithro721** [AntonyZ89/docker-qemu-arm#4] [General]
I installed directly in termux using ./termux-setup.sh
Docker was installed successfully 
Thn I installed portainer in it and I was able to access web ui using browser through 127.0.0.1:9000

Thn I installed prowlarr but I was not able to access using browser through 127.0.0.1:9695

**Comment by AntonyZ89** [comment]
You need to add all ports you want to use on the VM to the starqemu.ssh command:

```bash
qemu-system-x86_64 -machine q35 -m 1024 -smp cpus=2 -cpu qemu64 \
  -drive if=pflash,format=raw,read-only=on,file=$PREFIX/share/qemu/edk2-x86_64-code.fd \
  -netdev user,id=n1,\
net=192.168.50.0/24,\
hostfwd=tcp::2222-:22,\
hostfwd=tcp::9000-:9000,\
hostfwd=tcp::9695-:9695\ # new port
  -device virtio-net,netdev=n1\
  -nographic alpine.img
```
